### PR TITLE
Add logic to handle secondary actions and multi selecting to browse sdk.

### PR DESCRIPTION
**Pull request by doncung**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @doncung has signed the CLA. Thanks for the pull request!

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java [hunk 1]
```diff
@@ -304,11 +328,42 @@ public void onDetach() {
...
+    }
+
+
+
     protected void onInfoFetched(Intent intent) {
         onItemsFetched(intent);
     }
 
-    /**
+    /**P
```
Comment (brianytang) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java]: Remove

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/ThumbnailManager.java [hunk 2]
```diff
@@ -150,192 +227,23 @@ private int getDefaultIconResource(final BoxItem boxItem) {
             BoxFolder boxFolder = (BoxFolder) boxItem;
             if (boxFolder.getHasCollaborations() == Boolean.TRUE) {
                 if (boxFolder.getIsExternallyOwned() == Boolean.TRUE) {
-                    return R.drawable.ic_box_browsesdk_folder_external;
+                    return R.drawable.ic_box_browsesdk_locked;
                 }
                 return R.drawable.ic_box_browsesdk_folder_shared;
             }
-            return R.drawable.ic_box_browsesdk_folder;
+            return R.drawable.ic_box_browsesdk_personal;
```
Comment (brianytang) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/ThumbnailManager.java]: Can we be more descriptive with the icon names? locked -> folder_external and personal -> folder_personal, linked-> weblink?

**Comment by brianytang** [comment]
Is there a way to disable multi select from the fragment?

**Comment by brianytang** [comment]
LGTM +1

**Pull request merged**